Feat(content): Remnant: Scent of Rain 1 (Korath Tubfalet intro) (#7218)
endless-sky/endless-sky: data/remnant/remnant 2 side missions.txt

			`	Her gestures take on an apologetic tone. "It will not be a perfect replica as we will have to use our own systems for most of the internals, and the resource allotment cost will be significantly higher than it would have been back at the Tycho Crater shipyard on Luna... but if you have the resources and interest, the opportunity is there. The shipyard will have more information for you in a day or two." Her gestures trail off with a sign that indicates vast possibilities.`
			`	"Someday, Captain <first>, much may come from this. But for now, it will be months before we have analyzed everything. Thank you for helping recover a piece of our history, Captain."`

mission "Remnant: Scent of Rain 1"
	landing
	name "Search for the unknown Korath ship."
	description "Chilia has asked you to find the Korath ship that can carry three 'olofez."
	source
		government "Remnant"
	destination "Caelian"
	to offer
		has "Remnant: Cognizance 25: done"
	npc "scan outfits"
		government "Korath"
		personality waiting target
		system "Kor Zena'i"
		ship "Tubfalet" "Ra'puk At 295"

> 		dialog `In poor lighting, this ship might have passed for a Palavret, but major structural differences suggest this is at least a different model, or perhaps an entirely different ship. Time to report this to Chilia.`

Reviewer: This dialog comes up literally just after you scan it. Are these structural differences picked out by eye or the outfit scanner? It may be nice for once to see the player seeing something else other than lists of stuff in their scans (even if it is just in universe and not mechanically done)

Author: I made a slight tweak to this dialog to make it clear that it is scanning that reveals the structural differences.

Reviewer 2: The model does have a fair bit of visible varying structures that the base Raider does not have. It's not so much internal as it is simply the base of the ship itself being different visually (one extra asymmetrical bay, an asymmetrical gun pod, additional meat by the firelight ports, additional color rings, additional crystal-like structures, etc..

Author: Exactly. This dialog is making the point that all that visual stuff isn't just surface stuff - it is symptomatic of deep internal changes.

	npc
		government "Korath"
		personality waiting
		system "Kor Zena'i"
		fleet
			names "korath"
			cargo 3
			commodities Food Clothing Metal Plastic Medical "Heavy Metals"
			outfitters "Korath Basics" "Korath Exiles"
			variant
				"'olofez" 3
	on offer
		event "remnant: tubfalet appearance" 20 31
		conversation
			`You step out of your airlock to see Prefect Chilia striding towards you. "Captain <first>! I read a report mentioning that you dealt with a trio of 'olofez out in the Postverta region. Would you be interested in doing some reconnaissance for us? We would like to track down where those came from."`
			choice
				`	"Yes, I could do that."`
				`	"I would rather not."`
					decline

> 			`	He signs appreciatively. "We appreciate your willingness to help." He pulls up a hologram of the Ember Waste, and it shifts to display the systems towards the core. "Based on the Korath's typical movement patterns, we suspect that the ship should be somewhere in the nearer Korath systems. We will upload an algorithm to your targeting system to look for ship profiles that have no matches in our database. Please do an outfit scan of whatever ship you find."`

Reviewer: A lot of "appreciate" here... perhaps use a different word in the dialog?
```suggestion
			`	He signs appreciatively. "We admire your willingness to help." He pulls up a hologram of the Ember Waste, and it shifts to display the systems towards the core. "Based on the Korath's typical movement patterns, we suspect that the ship should be somewhere in the nearer Korath systems. We will upload an algorithm to your targeting system to look for ship profiles that have no matches in our database. Please do an outfit scan of whatever ship you find."`
```

Author: I changed other stuff before I got to this, so I couldn't apply it. But I have added it in manually. Thank you!

				accept
	on enter "Kor Zena'i"
		dialog

> 			`	The algorithm quickly parses the ship signatures present here, and notifies you that one in particular does not match its database: the one identified as the Ra'puk At 295. This must be the ship that Chilia would like a scan of.`

Reviewer: It feels as if the systems are quoting a ship name here? Often I find when referring to something unknown people put their declared names in quotation marks. This one isn't exactly important though, could work either way.
```suggestion
			`	The algorithm quickly parses the ship signatures present here, and notifies you that one in particular does not match its database: the one identified as the "Ra'puk At 295". This must be the ship that Chilia would like a scan of.`
```

	on complete
		payment 150000
		conversation
			`Chilia is waiting for you when you land. He looks over the scan results, nodding thoughtfully, and listens to your account of visiting Kor Zena'i.`
			`	"Thank you for investigating this, <first>. I will ensure that our engineers get your data immediately, and brief our patrols to be aware of this increased capacity." His gestures have a tone of anticipation and regret.`
			choice
				`	"You are welcome."`
					goto scanningpayment
				`	"Are you expecting something?"`
			`	Chilia smiles wistfully. "Do you know that feeling, when the whole world is still and the skies are clear, and yet you know in your bones that a storm is coming?" He continues without waiting for a response. "Well, I have had that feeling for decades, growing ever stronger. And now..." He pauses, drawing a deep breath. "Now, it feels like this is the first scent of rain. The first thing I can actually identify as a solid indicator that a storm is brewing."`
			label scanningpayment

> 			`	"But no matter. Planning for the storm is my duty, not yours, and the front is yet to appear. We have allocated you <payment> for the risks undertaken. Please use it wisely. While the overall storm is my duty to anticipate, it is your duty to prepare yourself for the wind." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`

Reviewer: This line doesn't make sense if you skip over Chilia's storm explanation and just say `"You are welcome."`
it comes out as `"You are welcome." "But no matter, Planning for the storm is my duty, ...."` which doesn't make much sense?
I'd create a new scanningpayment label that skips the storm to just pay you and place `decline` (to prematurely end the conversation) at the end of this section.

Author: I have adjusted the break point a bit so that it makes more sense. Thanks for pointing this out!

			`	Taely gestures take a tone of curiosity. "One thing that you could help us with, if you wish, is resources. Right now we do not have any real need to put all this machinery to work, and as such we do not have the resources to spare to get it fully operational. That being said, if ever you would like a replica of the Shattered Light, we could use this equipment to build one."`
			`	Her gestures take on an apologetic tone. "It will not be a perfect replica as we will have to use our own systems for most of the internals, and the resource allotment cost will be significantly higher than it would have been back at the Tycho Crater shipyard on Luna... but if you have the resources and interest, the opportunity is there. The shipyard will have more information for you in a day or two." Her gestures trail off with a sign that indicates vast possibilities.`
			`	"Someday, Captain <first>, much may come from this. But for now, it will be months before we have analyzed everything. Thank you for helping recover a piece of our history, Captain."`

mission "Remnant: Scent of Rain 1"
	landing
	name "Search for the unknown Korath ship."

> 	description "Chilia has asked you to find the Korath ship that can carry three 'olofez."

Reviewer 3: It occurred to me that my previous suggestion of waypointing the three Exile systems would be a bad idea since it'd require visiting all three to complete this without some shenanigans with invisible missions and stuff. This description should give some indication as to where the player should look. It should probably also mention that we need to do an outfit scan. Maybe:
```suggestion
	description "Chilia has asked you to find and conduct an outfit scan of the Korath ship that can carry three 'olofez. The Remnant believe it will be found in one of the southern Korath systems."
```
This format also feels off. I feel like we usually get mission descriptions telling us what to do, possibly with extra information like who asked us to do it. This is telling us that someone has asked us to do something. I don't know how consistent the rest of the game is on that, though.

	source

> 		government "Remnant"

Reviewer 3: This could offer on Ssil Vida but the conversation seems to assume we're not there.
`I read a report mentioning that you dealt with a trio of 'olofez out in the Postverta region.`

Author: Hmmmm, I think that's fine to say. Someone in California could tell someone else "So, I hear you have been starting businesses in the California region. Might be slightly odd turn of phrase, but no more so than, say, someone speaking in the third person or something. By which I mean, perfectly acceptable but different.

	destination "Caelian"
	to offer
		has "Remnant: Cognizance 25: done"
	on offer
		event "remnant: tubfalet appearance" 20 31
		conversation

> 			`You step out of your airlock to see Prefect Chilia striding towards you. "Captain <first>! I read a report mentioning that you dealt with a trio of 'olofez out in the Postverta region. Would you be interested in doing some reconnaissance for us? We would like to track down where those came from."`

Reviewer 4: After you say you can help. Chilia says "the ship should be somewhere in the nearer Korath systems." What ship? No ship is ever mention on this line. Perhaps add something about how they suspect that there is a single ship that carried those three 'olofezs there. That is, if it's a single ship with three bays then you know it's new.

Author: Expanded this along the lines of what you suggested.

			choice
				`	"Yes, I could do that."`
				`	"I would rather not."`

> 					decline

Reviewer 4: Maybe follow this up with one line from Chilia before declining instead of ending the mission immediately on the choice.

Author: Alright, added a line here.

			`	He signs appreciatively. "We admire your willingness to help." He pulls up a hologram of the Ember Waste, and it shifts to display the systems towards the core. "Based on the Korath's typical movement patterns, we suspect that the ship should be somewhere in the nearer Korath systems. We will upload an algorithm to your targeting system to look for ship profiles that have no matches in our database. Please do an outfit scan of whatever ship you find."`
				accept
	npc "scan outfits"
		government "Korath"

> 		personality waiting target

Reviewer 4: Should this be staying, not waiting? If not, the "waiting" personality doesn't do anything here. Given that there's an on enter for this NPC's spawn system, I imagine the answer is yes.
```suggestion
		personality staying target
```

		system "Kor Zena'i"
		ship "Tubfalet" "Ra'puk At 295"

> 		dialog `In poor lighting, this ship might have passed for a Palavret, but scanning reveals major structural differences. These suggest this is at least a different model, or perhaps an entirely different ship. Time to report this to Chilia.`

Reviewer 3: ```suggestion
		dialog `In poor lighting, this ship might have passed for a Palavret, but major structural differences suggest this is at least a modified design, or perhaps an entirely different ship. Time to report this to Chilia.`
```
`different model` and `entirely different ship` don't seem like meaningfully different terms. Alternatively:
```suggestion
		dialog `In poor lighting, this ship might have passed for a Palavret, but major structural differences suggest this is at least a new generation of the model, or perhaps an entirely different ship. Time to report this to Chilia.`
```

> 		dialog `In poor lighting, this ship might have passed for a Palavret, but scanning reveals major structural differences. These suggest this is at least a new generation of the model, or perhaps an entirely different ship. Time to report this to Chilia.`

Reviewer 3: ```suggestion
		dialog `In poor lighting, this ship might have passed for a Palavret, but scanning reveals major structural differences. These suggest this is at least a new generation of the model, or perhaps an entirely different design. Time to report this to Chilia.`
```
Maybe

	npc
		government "Korath"

> 		personality waiting

Reviewer 4: Ditto, but doesn't matter as much since these are fighters that can't leave anyway.
```suggestion
		personality staying
```

		system "Kor Zena'i"
		fleet
			names "korath"
			cargo 3
			commodities Food Clothing Metal Plastic Medical "Heavy Metals"
			outfitters "Korath Basics" "Korath Exiles"
			variant
				"'olofez" 3
	on enter "Kor Zena'i"
		dialog

> 			`	The algorithm quickly parses the ship signatures present here, and notifies you that one in particular does not match its database: the one identified as the "Ra'puk At 295". This must be the ship that Chilia would like a scan of.`

Reviewer 4: ```suggestion
			`The algorithm quickly parses the ship signatures present here, and notifies you that one in particular does not match its database: the one identified as the "Ra'puk At 295." This must be the ship that Chilia would like a scan of.`
```

	on complete
		payment 150000
		conversation

> 			`Chilia is waiting for you when you land. Judging by the hologram floating before him, he appears to already be reviewing the data from your scan. As you approach, he pushes it to the side to focus on you. "This is interesting data." He comments. "What are your thoughts, having seen it?" He listens attentively to your account of visiting Kor Zena'i, nodding thoughtfully at various insights.`

Reviewer 3: ```suggestion
			`Chilia is waiting for you when you land. Judging by the hologram floating before him, he appears to already be reviewing the data from your scan. As you approach, he pushes it to the side to focus on you. "This is interesting data," he comments. "What are your thoughts, having seen it first hand?" He listens attentively to your account of visiting Kor Zena'i, nodding thoughtfully at various insights.`
```

> 			`Chilia is waiting for you when you land. Judging by the hologram floating before him, he appears to already be reviewing the data from your scan. As you approach, he pushes it to the side to focus on you. "This is interesting data," he comments. "What are your thoughts, having seen it first hand?" He listens attentively to your account of visiting Kor Zena'i, nodding thoughtfully at various insights.`

Reviewer 4: If you're going to have a character ask for the player's thoughts but then not even give a choice, you might as well not even have asked in the first place. That is, either remove Chilia asking you this question, or let the player respond to it.

Author: Add dialog choices. Not entirely happy with the options, to be honest, but they work.

			`	"Thank you for investigating this, <first>. I will ensure that our engineers get your data immediately, and brief our patrols to be aware of this increased capacity." His gestures have a tone of anticipation and regret.`
			choice
				`	"You are welcome."`
					goto scanningpayment
				`	"Are you expecting something?"`
			`	Chilia smiles wistfully. "Do you know that feeling, when the whole world is still and the skies are clear, and yet you know in your bones that a storm is coming?" He continues without waiting for a response. "Well, I have had that feeling for decades, growing ever stronger. And now..." He pauses, drawing a deep breath. "Now, it feels like this is the first scent of rain. The first thing I can actually identify as a solid indicator that a storm is brewing."`

> 			`	He shakes his head "But no matter. Planning for the storm is my duty, not yours, and the front is yet to appear.`

Reviewer 4: ```suggestion
			`	He shakes his head. "But no matter. Planning for the storm is my duty, not yours, and the front is yet to appear.`
```

			label scanningpayment

> 			`	"We have allocated you <payment> for the risks undertaken. Please use it wisely. While the overall threats are my duty to anticipate, it is your duty to prepare yourself for the wind." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`

Reviewer: Just one little note, would `"wind"` be appropriate here still? I know that you have gone through this entire conversation but just pointing this out as it looks like you might have missed it. Just this one comment this time.
```suggestion
			`	"We have allocated you <payment> for the risks undertaken. Please use it wisely. While the overall threats are my duty to anticipate, it is your duty to prepare yourself for the struggle." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`
```

> 			`	"We have allocated you <payment> for the risks undertaken. Please use it wisely. While the overall threats are my duty to anticipate, it is your duty to prepare yourself for the wind." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`

Reviewer 3: `overall threats` here makes it seem like this is a poor translation. And, I agree with Pilover that the use of `wind` like this doesn't seem to make a whole lot of sense, or even have a clear meaning, if a person skipped the storm symbolism before it.
```suggestion
			`	"We have allocated you <payment> for the risks undertaken. Please use it wisely. While it is my duty to anticipate threats to us all, it is your duty to ensure you are prepared to face the winds they may bring." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`
```

			`	Taely gestures take a tone of curiosity. "One thing that you could help us with, if you wish, is resources. Right now we do not have any real need to put all this machinery to work, and as such we do not have the resources to spare to get it fully operational. That being said, if ever you would like a replica of the Shattered Light, we could use this equipment to build one."`
			`	Her gestures take on an apologetic tone. "It will not be a perfect replica as we will have to use our own systems for most of the internals, and the resource allotment cost will be significantly higher than it would have been back at the Tycho Crater shipyard on Luna... but if you have the resources and interest, the opportunity is there. The shipyard will have more information for you in a day or two." Her gestures trail off with a sign that indicates vast possibilities.`
			`	"Someday, Captain <first>, much may come from this. But for now, it will be months before we have analyzed everything. Thank you for helping recover a piece of our history, Captain."`

mission "Remnant: Scent of Rain 1"

> 	name "Search for the unknown Korath ship."
> 	description "Chilia has asked you to find the Korath ship that can carry three 'olofez."
> 	landing

Reviewer 3: ```suggestion
	landing
	name "Search for the unknown Korath ship."
	description "Chilia has asked you to find the Korath ship that can carry three 'olofez."
```

	source
		government "Remnant"
	destination "Caelian"
	to offer
		has "Remnant: Cognizance 25: done"
	npc "scan outfits"
		government "Korath"
		personality waiting target

> 		system
> 			system "Kor Zena'i"

Reviewer 3: ```suggestion
		system "Kor Zena'i"
```

		ship "Tubfalet" "Ra'puk At 295"

> 		dialog `In poor lighting this ship might have passed for a Palavret, but major structural difference suggest this is at least a different model, perhaps an entirely different ship. Time to report this to Chilia.`

Reviewer 5: ```suggestion
		dialog `In poor lighting this ship might have passed for a Palavret, but major structural difference suggest this is at least a different model, or perhaps an entirely different ship. Time to report this to Chilia.`
```

> 		dialog `In poor lighting this ship might have passed for a Palavret, but major structural difference suggest this is at least a different model, or perhaps an entirely different ship. Time to report this to Chilia.`

Reviewer 3: ```suggestion
		dialog `In poor lighting, this ship might have passed for a Palavret, but major structural differences suggest this is at least a different model, or perhaps an entirely different ship. Time to report this to Chilia.`
```
`this is at least a different model, or perhaps an entirely different ship`
What does this mean? It seems like a tautology.

Author: What I was going for is conveying that the pilot isn't really sure what it is: Seen at a glance, this ship could be mistaken for a Palavret. But with the scans the structural detail makes it clear that this is at least a different version of the ship that we have seen to date, maybe even a completely new ship that just happens to have a strong surface resemblence to the Palavret.

Reviewer 3: ```suggestion
		dialog `In poor lighting, this ship might have passed for a Palavret, but major structural differences suggest this is at least a modified design, or perhaps an entirely different ship. Time to report this to Chilia.`
```
`different model` and `entirely different ship` don't seem like meaningfully different terms. Alternatively:
```suggestion
		dialog `In poor lighting, this ship might have passed for a Palavret, but major structural differences suggest this is at least a new generation of the model, or perhaps an entirely different ship. Time to report this to Chilia.`
```

	npc
		government "Korath"
		personality waiting

> 		system
> 			system "Kor Zena'i"

Reviewer 3: ```suggestion
		system "Kor Zena'i"
```

		fleet
			names "korath"
			cargo 3
			commodities Food Clothing Metal Plastic Medical "Heavy Metals"
			outfitters "Korath Basics" "Korath Exiles"
			variant
				"'olofez" 3

> 	on offer

Reviewer 3: This should be above the `npc`s.

Author: Moved. Thanks!

> 		event "remnant: tubfalet appearance" 20 31

Reviewer 3: It seems like it'd be possible to just not complete this mission until this event happens, at which point, it seems a bit silly to still require us to scan one particular ship, especially when they will be venturing all the way to the Remnant home systems on raids. This mission should probably just have a deadline or otherwise end before we get to this event.

Reviewer 3: It might be a good idea to mention that in the `on complete` conversation, then, instead of having him act like this is the first scan he's seen of one of these ships.

Author: Well, with the description as written, it fits several situations:
1. This is the first good scan the Remnant have received of it.
2. Chilia is being polite and/or respectful of the pilot's efforts (like any good military commander: If a long range scouts goes above and beyond scouting the enemy's forces and brings back detailed imagery of a new weapon, they listen and read the report and appreciate it - not just going "oh, I already got a report on that.")
3. Chilia doesn't want to reveal that they already have scans of it.

The main situation it *doesn't* accommodate is:
4. The Remnant already have scans, and want to either show off to the player or "put them in their place" by revealing that, no matter how fast the player got them, the Remnant *already* have scans.

In this particular situation, 1 & 2 are the primary alternatives I want to account for. I do *not* want to risk implying 4 here.

Author: > It might be a good idea to mention that in the `on complete` conversation, then, instead of having him act like this is the first scan he's seen of one of these ships.

@warp-core I didn't actually state as much in my previous post, but I did adjust the on complete to show that Chilia is looking at a scan before the player arrives. He doesn't elaborate on whether that's the pilot's data transmitted ahead, or from another source. Leaving it imprecise can let the pilot imagine whatever fits best to them. Does this work for you?

		conversation
			`You step out of your airlock to see Prefect Chilia striding towards you. "Captain <first>! I read a report mentioning that you dealt with a trio of 'olofez out in the Postverta region. Would you be interested in doing some reconnaissance for us? We would like to track down where those came from."`
			choice
				`	"Yes, I could do that."`
				`	"I would rather not."`
					decline

> 			`	He signs appreciatively. "We appreciate your willingness to help." He pulls up a hologram of the Ember Waste, and it shifts to display the systems towards the core. "Based on the Korath's typical movement patterns, we suspect that the ship should be somewhere in the nearer Korath systems. We will upload an algorithm to your targeting system to look for ship profiles that have no matches in our database. Please do an outfit scan of whatever ship you find."`

Reviewer 3: ```suggestion
			`	He signs his thanks, "We appreciate your willingness to help." He pulls up a hologram of the Ember Waste, and it shifts to display the systems towards the core. "Based on the Korath's typical movement patterns, we suspect that the ship will be somewhere in the nearer Korath systems. We will upload an algorithm to your targeting system to look for ship profiles that have no matches in our database. Please do an outfit scan of whatever ship you find."`
```
Do they think I'm too stupid to spot the one that's different myself?
Also, the mission should probably waypoint the three nearer systems.

Author: a) Not all players hang out in exile systems and are familiar with what to expect there.
b) Korath territory is a bit more active with PRs lately, so stuff is changing there, and even people that *were* quite familiar with the area 2-3 years ago may find it less so once 0.9.16 roles around.
c) It's an in-game lore explanation for why your scanner highlights a ship with the "target" behavior.
d) It makes sense. They've *got* extensive databases of Korath ships, it makes a lot of sense that if they're asking you to scan the different one, that they would give you a signature database to let your systems automatically pick out the different one.

For the dialog suggestion itself: the "He signs appreciatively" is meant to be referring to his tone; I've been deliberately incorporating a range of tones and meanings into the Remnant sign language.

As for the tense should vs will... I'm good with that, although I was orginally thinking that he'd be talking about where it is now, rather than predicting where it will be in the future.

Reviewer 6: I'd go with should instead, tbfh.

				accept
	on enter "Kor Zena'i"
		dialog

> 			`	The algorithm quickly parses the ship signatures present here, and notifies you that one in particular does not match its database: The one identified as the Ra'puk At 295. This would be the ship that Chilia would like a scan of.`

Reviewer 5: ```suggestion
			`	The algorithm quickly parses the ship signatures present here, and notifies you that one in particular does not match its database: the one identified as the Ra'puk At 295. This would be the ship that Chilia would like a scan of.`
```

> 			`	The algorithm quickly parses the ship signatures present here, and notifies you that one in particular does not match its database: the one identified as the Ra'puk At 295. This would be the ship that Chilia would like a scan of.`

Reviewer 3: ```suggestion
			`	The algorithm quickly parses the ship signatures present here, and notifies you that one in particular does not match its database: the one identified as the Ra'puk At 295. This must be the ship that Chilia would like a scan of.`
```
```suggestion
			`	The algorithm quickly parses the ship signatures present here, and notifies you that one in particular does not match its database: the one identified as the Ra'puk At 295. This is the ship that Chilia would like a scan of.`
```

	on complete
		payment 150000
		conversation

> 			`Chilia is waiting for you when you land. He looks over the scan results, nodding thoughtfully, and listens to your account of visiting Kor Zena'i.`

Reviewer 3: Why do I need to bring the scan back to him at all?
In that other mission, someone said something about a `Quantum Entangled Network`. Why can't the scan data just be transmitted along that? He should've seen this information a week ago.
Maybe he wants my thoughts on it or otherwise wants to discuss it in person, but the data could at least be accessed sooner.

Author: This section has been updated to make it clear that Chilia already has the data, and is interested in what *you* have to say about the ship. Thanks for bringing this up!

			`	"Thank you for investigating this, <first>. I will ensure that our engineers get your data immediately, and brief our patrols to be aware of this increased capacity." His gestures have a tone of anticipation and regret.`
			choice
				`	"You are welcome."`
					goto scanningpayment
				`	"Are you expecting something?"`

> 			`	Chilia smiles wistfully. "Do you know that feeling, when the whole world is still and the skies are clear, and yet you know in your bones that a storm is coming?" He continues without waiting for a response. "Well, I have had that feeling for decades, growing ever stronger. And now..." He pauses, drawing a deep breath. "Now, it feels like this is the first scent of rain. The first thing I can actually identify as a solid indicator that a storm is brewing."`

Reviewer 3: You feel the air is [charged, tense] like the moment before a storm? Seems that's going around.
Though, honestly, if you've had that feeling for decades, you're probably just paranoid. Even if the "storm" does come, at this point, it's just a broken clock being right twice a day. You should speak to professional, Chilia.

Author: hehe. Yeah, probably...

			label scanningpayment

> 			`	"But no matter. Planning for the storm is my duty, not yours, and the stormfront is yet to appear. We have allocated you <payment> for the risks undertaken, please use it wisely: While the overall storm is my duty to anticipate, it is your duty to prepare yourself for the wind." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`

Reviewer 5: ```suggestion
			`	"But no matter. Planning for the storm is my duty, not yours, and the stormfront is yet to appear. We have allocated you <payment> for the risks undertaken. Please use it wisely. While the overall storm is my duty to anticipate, it is your duty to prepare yourself for the wind." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`
```

> 			`	"But no matter. Planning for the storm is my duty, not yours, and the stormfront is yet to appear. We have allocated you <payment> for the risks undertaken. Please use it wisely. While the overall storm is my duty to anticipate, it is your duty to prepare yourself for the wind." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`

Reviewer 3: Dude seems obsessed with storms. Can't go five words without saying `storm`.
```suggestion
			`	"But no matter. Planning for the storm is my duty, not yours, and the front is yet to appear. We have allocated you <payment> for the risks undertaken. Please use it wisely. While the overall storm is my duty to anticipate, it is your duty to prepare yourself for the wind." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`
```
`it is your duty to prepare yourself for the wind`
What part of this do you want to emphasise?

Author: Ummm... In my head it doesn't really have an emphasis point. But probably *your*

			choice
				`	"Yes, I could do that."`
				`	"I would rather not."`
					goto rathernot
			`	He signs appreciatively. "We admire your willingness to help." He pulls up a hologram of the Ember Waste, and it shifts to display the systems towards the core. "We are concerned that the Korath may have a new ship that can carry three fighters. Based on the Korath's typical movement patterns, we suspect that - if such a ship exists - it should be somewhere in the nearer Korath systems. We will upload an algorithm to your targeting system to look for ship profiles that have no matches in our database. Please do an outfit scan of whatever ship you find."`
				accept
			label rathernot
			`	He nods respectfully. "Well, thank you for all that you have done for us. I will see who else is available." With that, he heads off into the starport.`
				decline
	npc "scan outfits"
		government "Korath"
		personality staying target
		system "Kor Zena'i"

> 		ship "Tubfalet" "Ra'puk At 295"

Reviewer 7: ```suggestion
		ship "Tubfalet" "Ra'puk At 333"
```

		dialog `In poor lighting, this ship might have passed for a Palavret, but scanning reveals major structural differences. These suggest this is at least a new generation of the model, or perhaps an entirely different design. Time to report this to Chilia.`
	npc
		government "Korath"
		personality staying
		system "Kor Zena'i"
		fleet
			names "korath"
			cargo 3
			commodities Food Clothing Metal Plastic Medical "Heavy Metals"
			outfitters "Korath Basics" "Korath Exiles"
			variant
				"'olofez" 3
	on enter "Kor Zena'i"
		dialog

> 			`The algorithm quickly parses the ship signatures present here, and notifies you that one in particular does not match its database: the one identified as the "Ra'puk At 295." This must be the ship that Chilia would like a scan of.`

Reviewer 7: ```suggestion
			`The algorithm quickly parses the ship signatures present here, and notifies you that one in particular does not match its database: the one identified as the "Ra'puk At 333." This must be the ship that Chilia would like a scan of.`
```

			`	Taely gestures take a tone of curiosity. "One thing that you could help us with, if you wish, is resources. Right now we do not have any real need to put all this machinery to work, and as such we do not have the resources to spare to get it fully operational. That being said, if ever you would like a replica of the Shattered Light, we could use this equipment to build one."`
			`	Her gestures take on an apologetic tone. "It will not be a perfect replica as we will have to use our own systems for most of the internals, and the resource allotment cost will be significantly higher than it would have been back at the Tycho Crater shipyard on Luna... but if you have the resources and interest, the opportunity is there. The shipyard will have more information for you in a day or two." Her gestures trail off with a sign that indicates vast possibilities.`
			`	"Someday, Captain <first>, much may come from this. But for now, it will be months before we have analyzed everything. Thank you for helping recover a piece of our history, Captain."`

mission "Remnant: Scent of Rain 1"
	landing

> 	name "Search for the unknown Korath ship."

Reviewer 4: Names aren't sentences
```suggestion
	name "Search for the unknown Korath ship"
```

> 	description "Chilia has asked you to find and conduct an outfit scan of the Korath ship that can carry three 'olofez, then report back to <destination>. The Remnant believe it will be found in one of the southern Korath systems."

Reviewer 7: ```suggestion
	description "Chilia has asked you to find and conduct an outfit scan of the Korath ship that can carry three 'olofezes, then report back to <destination>. The Remnant believe it will be found in one of the southern Korath systems."
```

	source
		government "Remnant"
	destination "Caelian"
	to offer
		or
			and
				has "Remnant: Cognizance 25: done"
				not "flagship planet: Ssil Vida"
			has "event: remnant: cognizance calm"
	on offer
		event "remnant: tubfalet appearance" 20 31
		conversation
			branch scentssilvida
				has "flagship planet: Ssil Vida"

> 			`You step out of your airlock to see Prefect Chilia striding towards you. "Captain <first>! I read a report mentioning that you dealt with a trio of 'olofez out in the Postverta region. Would you be interested in doing some reconnaissance for us? We would like to track down where those came from."`

Reviewer 7: ```suggestion
			`You step out of your airlock to see Prefect Chilia striding towards you. "Captain <first>! I read a report mentioning that you dealt with a trio of 'olofezes out in the Postverta region. Would you be interested in doing some reconnaissance for us? We would like to track down where those came from."`
```

				goto scentrainchoice
			label scentssilvida

> 			`You step out of your airlock to see Prefect Chilia striding towards you. "Captain <first>! I read a report mentioning that you dealt with a trio of 'olofez patrolling alone near here. Would you be interested in doing some reconnaissance for us? We would like to track down where those came from."`

Reviewer 7: ```suggestion
			`You step out of your airlock to see Prefect Chilia striding towards you. "Captain <first>! I read a report mentioning that you dealt with a trio of 'olofezes patrolling alone near here. Would you be interested in doing some reconnaissance for us? We would like to track down where those came from."`
```

		system "Kor Zena'i"
		ship "Tubfalet" "Ra'puk At 333"
		dialog `In poor lighting, this ship might have passed for a Palavret, but scanning reveals major structural differences. These suggest this is at least a new generation of the model, or perhaps an entirely different design. Time to report this to Chilia.`
	npc
		government "Korath"
		personality staying
		system "Kor Zena'i"
		fleet
			names "korath"
			cargo 3
			commodities Food Clothing Metal Plastic Medical "Heavy Metals"
			outfitters "Korath Basics" "Korath Exiles"
			variant
				"'olofez" 3
	on enter "Kor Zena'i"
		dialog
			`The algorithm quickly parses the ship signatures present here, and notifies you that one in particular does not match its database: the one identified as the "Ra'puk At 333." This must be the ship that Chilia would like a scan of.`
	on complete
		payment 150000
		conversation
			`Chilia is waiting for you when you land. Judging by the hologram floating before him, he appears to already be reviewing the data from your scan. As you approach, he pushes it to the side to focus on you. "This is interesting data," he comments. "What are your thoughts, having seen it first hand?"`
			choice

> 				`	(Describe the addition of the new bay and how it is used.)`
> 				`	(Describe the structural changes and potential ways that could change the ship's performance.)`

Reviewer 4: If the player is saying something then actually say something. These choices are longer than the usual, but I think fit the situation well in having the player actually describe the new ship.
```suggestion
				`	"You were right in suspecting that this new ship carries an additional fighter. One more fighter might not change much in combat, but if the Korath are building new warships, they may also be building stronger fighters."`
				`	"An extra fighter bay isn't the only thing this new ship has. It also comes with an extra gun, bigger engines, superior shields and hull, and likely other unseen improvements. Overall it's much more formidable than its predecessor."`
```

> 			`He listens attentively to your account of visiting Kor Zena'i, nodding thoughtfully at various insights.`

Reviewer 4: ```suggestion
			`	He listens attentively to your account of visiting Kor Zena'i, nodding thoughtfully at various insights.`
```
